src/components/Header/NotificationButton.tsx

```diff
@@ -63,16 +64,17 @@ const NotificationButton = () => {
                   <p className="max-h-14 flex-1 overflow-hidden text-xl font-bold">
                     {message.content}
                   </p>
-                  <p className="ml-6 w-[84px] text-right text-sm">
+                  <p className="ml-6 w-[5rem] text-right text-sm font-medium leading-[0.875rem]">
                     {message.time}
                   </p>
                 </div>
               )
           )}
-          <Separator className="h-[1px] bg-primary" />
+          <Separator className="h-[0.0625rem] bg-primary" />
           <Link
             href="/"
-            className="flex h-[4.5rem] w-full items-center justify-center"
+            ref={triggerRef as React.RefObject<HTMLAnchorElement>}
+            className="flex h-[4.5rem] w-full items-center justify-center hover:bg-primary/[0.03]"
           >
             查看所有通知
           </Link>
```

Comment:
似乎太複雜，應該不用`useEffect`和`useRef`去監聽是否關閉，改成用`useRouter`和`useState`就能達成。大概如下，應該就能達到:
```suggestion
          <Button
            variant="ghost"
            onClick={() => {
              setIsOpen((prev) => !prev);
            }}
            asChild
            className="flex h-[4.5rem] w-full items-center justify-center hover:bg-primary/[0.03]"
          >
            <Link href="/">查看所有通知</Link>
          </Button>
```

Reply:
或是更簡單的:
```suggestion
          <Link
            href="/"
            onClick={() => {
              setIsOpen((prev) => !prev);
            }}
            className="flex h-[4.5rem] w-full items-center justify-center"
          >
            查看所有通知
          </Link>
```